pool/connection_pool.go

```diff
@@ -1187,22 +1170,20 @@ func (p *ConnectionPool) fillPools(ctx context.Context,
 				instance.Name, err)
 			select {
 			case <-ctx.Done():
-				ctxCanceled = true
-
 				p.ends[instance.Name] = nil
 				log.Printf("tarantool: operation was canceled")
 
 				p.deactivateConnections()
 
-				return false, ctxCanceled
+				return true
 			default:
 			}
-		} else if p.processConnection(conn, instance.Name, end) {
-			somebodyAlive = true
+		} else {
+			p.processConnection(conn, instance.Name, end)
```

Comment:
Is there any reason for keeping the return value for `processConnection`? Seems like this is the only place we use this method, maybe some refactoring should be done? For example, just delete the return value.
https://github.com/tarantool/go-tarantool/blob/50dc5d0d9a93d7fabf0325014edd918321f04274/pool/connection_pool.go#L1135-L1136

Reply:
Thank you, updated!